### ISSUE TITLE
Unable to retrieve mac binary for latest release. v 1.14.3

### ISSUE THREAD

**Issue by gaganhegde** [operator-framework/operator-registry#487]
The latest release https://github.com/operator-framework/operator-registry/releases/tag/v1.14.3 , does not contain a mac binary, much unlike the previous releases.

Cloning the code locally and running the make build or go build ./cmd/opm throws an error.

```
/usr/local/Cellar/go/1.15.2/libexec/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: sectionForAddress(0x132097E) address not in any section file '/var/folders/ns/c2jtxl_s0s10rd20d8vg0zfm0000gn/T/go-link-033558814/go.o' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

```

**Comment by rporres** [comment]
v1.14.2 contains darwin-amd64-opm and linux-amd64-opm
v1.14.3 contains linux-ppc64le-opm and linux-amd64-opm

**Comment by benluddy** [comment]
We're working on fixing the inconsistent release automation now. Thanks for filing an issue.

The darwin linker error is a go 1.15 bug that apparently has a fix in place for 1.15.4. In the meantime, you should be able to build with go 1.14.

**Comment by benluddy** [comment]
The intended release binaries (linux/amd64, linux/ppc64le, linux/s390x, darwin/amd64, and windows/amd64) should all be present now in https://github.com/operator-framework/operator-registry/releases/tag/v1.15.1. We'll need to introduce better automated checks to prevent issues like this from happening again.
